test.sh

```diff
@@ -10,8 +9,9 @@ sqlite3 "$testdir/core.db" < sql/init_data/init_core.sql;
 sqlite3 "$testdir/common.db" < sql/init_data/init.sql;
 sqlite3 "$testdir/selectfunc.db" < sql/init_data/init_selectfunc.sql;
 
-sed -i 's/REGRESS =.*/REGRESS = extra\/sqlite_fdw_post extra\/bitstring extra\/bool extra\/float4 extra\/float8 extra\/int4 extra\/int8 extra\/numeric extra\/out_of_range extra\/timestamp extra\/uuid extra\/join extra\/limit extra\/aggregates extra\/prepare extra\/select_having extra\/select extra\/insert extra\/update extra\/encodings sqlite_fdw type aggregate selectfunc /' Makefile
-
+export "$1"; # this argument is usually a varibales with values
+[ -z "$ENABLE_GIS" ] && gispref='no' || gispref='post';
```

Comment:
`gispref` will be set if `export "$1";` cause `ENABLE_GIS=1`. This means `ENABLE_GIS=1` as first argument of the script.

Reply:
Now environment variable passing changed to outer `export` style without any processing in `test.sh`. Please review this aspect again, @son-phamngoc

Reply:
@mkgrgis Sorry, please do not change it to `export`. We would like to use the argument of script. It will be more flexible.  
Could you use argument of script?

Reply:
> Could you use argument of script?

No problems, please verify results of https://github.com/pgspider/sqlite_fdw/pull/96/commits/5f473bea5eff4cae2c308e4da4764466febf5edb. For me this works fine.

Reply:
Thank you. Confirmed. Please help me to resolve this comment.